app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -531,6 +531,31 @@ BrowserWorld::State::UpdateControllers(bool& aRelayoutWidgets) {
     if (controller.modelToggle)
       controller.modelToggle->ToggleAll(controller.mode == ControllerMode::Device);
 
+    if (controller.leftHandActionEnabled && controller.handActionButtonTransform != nullptr) {
+      // Layout the button between the thumb and index fingertips
+      const int OPENXR_HAND_JOINT_INDEX_TIP = 10;
+      const int OPENXR_HAND_JOINT_THUMB_TIP = 5;
```

Comment:
We should make it independent of the SDK. Maybe we could define an enum with the 26 hand joints from the OpenXR spec and hope for the best. Another option would be to ask the device delegate maybe...